ci/jenkins/test.sh

```diff
@@ -734,6 +748,15 @@ function run_conformance {
     kubectl rollout status daemonset/antrea-agent -n kube-system
 
     set +e
+    # Conformance dns test cases fail in ipv6 kind cluster because here CoreDNS pods are ipv6 pods and it can't reach ipv4 upstream namespace server.
+    # Note: By default CoreDNS ConfigMap uses /etc/resolv.conf to get upstream nameservers which can be accessed by ipv4 and single stack ipv6 CoreDNS would
+    #  not be able to connect that server and it return servfail, to reolve this we need to forward ipv6 query to "vmware.com:53" server and it will return 'NXDOMAIN' so that client keep searching.
```

Comment:
Please avoid using longer sentence in a single line. Use multiple lines of about 80 chars instead.